bloom/generators/debian/generator.py

```diff
@@ -471,6 +485,10 @@ def __process_template_folder(path, subs):
             os.path.relpath(item),
             os.path.relpath(template_path)))
         result = em.expand(template, **subs)
+        # Skip the result is empty
```

Comment:
Looks like a small word omission typo. Perhaps change it to "Skip if the result is empty" or "Don't write an empty file"?